include/clang/Interpreter/CppInterOp.h

```diff
@@ -88,6 +88,12 @@ namespace Cpp {
 
   enum OperatorArity { kUnary = 1, kBinary, kBoth };
 
+  enum Qualifier : unsigned {
+    Const = 0b01,
+    Volatile = 0b010,
+    Restrict = 0b0100
```

Comment:
```suggestion
    Const = 0,
    Volatile = 1 << 0, // 1
    Restrict = 1 << 1 // 2
```

or just write 1 and 2.